examples/Huggingface_Transformers/README.md

```diff
@@ -58,6 +58,8 @@ In the setup_config.json :
 
 *model_name* : bert-base-uncased , roberta-base or other available pre-trained models.
 
+*fine_tuned* : name or path to the finetuned model if available, for example "distilbert-base-cased-distilled-squad", leave it blank if not available.
```

Comment:
There is already a model_name field. Can that not be used for fine tuned models name/path for the finetuned models as well? Please explain why we need a second field.

Reply:
In the current setting , when we do not pass the vocab.txt as the artifacts, tokenizer uses the pretrained version as being passed by Model_name. In that case fine_tuned version can not be used for the tokenizer.